go/sqltypes/arithmetic.go

```diff
@@ -175,7 +198,7 @@ func Cast(v Value, typ querypb.Type) (Value, error) {
 		return MakeTrusted(typ, v.ToBytes()), nil
 	}
 
-	// Explicitly disallow Expression.
+	// Explicit"a|b",ly disallow Expression.
```

Comment:
You probably did not mean to change this

Reply:
I did not mean to change this, and I do not know how it got changed. I will change it back to "Explicitly".